src/main/kotlin/org/rust/cargo/project/CargoProjectDescription.kt

```diff
@@ -102,7 +100,7 @@ class CargoProjectDescription private constructor(
 
     fun isCrateRoot(file: VirtualFile): Boolean = findTargetForCrateRootFile(file) != null
 
-    fun withAdditionalPackages(additionalPackages: Collection<Pair<String, VirtualFile>>): CargoProjectDescription {
+    fun withAdditionalPackages(additionalPackages: Collection<Pair<String, VirtualFile>>, origin: PackageOrigin): CargoProjectDescription {
```

Comment:
This function has only usage, inside a function that has only one usage... Perhaps we can rename this to `withStdlib`, remove `origin` argument and maybe even kill `extendProjectDescriptionWithStandardLibraryCrates` function?